### PR TITLE
Register cdn.lumix.is-a.dev

### PR DESCRIPTION
**Pull request by nyxaroo**
Register cdn.lumix.is-a.dev with CNAME record pointing to nas08452f.myqnapcloud.com.

### CODE REVIEW

File: domains/cdn.lumix.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "o-lumix",
+           "email": "",
+           "discord": "828313756827516949",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.DztjsgEqh20R7XRvy1Ys59WAt3E3Mop9FI8xUZSm8ySdopNznv1CPcXUHffF7l6qmZ0GLJrP1HOi7KmWWMo5dWfesA1ElFiRZ-w5ERAKqII9gN8vRJfT7U2puo05zDnb31r1sLpxRo4B4XoVFYheaaZV89I_xuPGUbRISh504GmsDy1efnmeM5s99Y0lBA8RPwOlJk5PaxAk-6pvpbP-VFjET7xPyqdkI36BAOHkoPcDW2z1nlnPRfqVx-_oVWWUxRjmrcDyWT8h6RIPTnohHulJy3Hcr3vNL9BArQcEEac8ldcMKJ7GNgFVoa_gw9-IzVjXuGVJGms3FQiQzJcy8w.R-8EwsLmUh6K87ptS8MC6Q.tf9kATvHd4W8qJvy05zVeb-qBqtyqG2Vp0mHr7MzwKwkHC6ujZc9-bGcAHE2bKk83IUZNTitDk2mMXgXEdV8J_XmrP6L_9dHzoE29wgUcyE.w5cX97lF0fmgwmsIvs5ZfQ"
+        },
+    
+        "record": {
+            "CNAME": "nas08452f.myqnapcloud.com"
```
Comment (DEV-DIBSTER) [inline domains/cdn.lumix.json:10]: Site doesn't load. Can you send a preview?

File: domains/sql.lumix.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "o-lumix",
+           "email": "lumixofficiel@gmail.com",
+           "discord": "828313756827516949"
+        },
+    
+        "record": {
+            "CNAME": "o-lumix.github.io/sql"
```
Comment (DEV-DIBSTER) [inline domains/sql.lumix.json:9]: Invalid CNAME record, as it should not contain file paths.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/cdn.lumix.json (hunk 1 above)
Comment (o-lumix) [reply domains/cdn.lumix.json:10]: The NAS is not turned on, I could turn it on tomorrow

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**